common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -210,12 +210,14 @@ def student_view(self, context):
         self._capture_basic_metrics()
 
         # Is this sequential part of a timed or proctored exam?
+        masquerading = context.get('specific_masquerade', False)
+        view_html = None
```

Comment:
I'd prefer `special` over `timed`, as this logic applies to proctored exams as well. The feature flag for enabling these types of exams is already `ENABLE_SPECIAL_EXAMS`